projects/plugins/boost/app/assets/src/js/pages/settings/elements/Module.svelte

```diff
@@ -1,74 +1,58 @@
 <script lang="ts">
 	import { createEventDispatcher, onMount } from 'svelte';
 	import Toggle from '../../../elements/Toggle.svelte';
-	import { modulesState, modulesStatePending } from '../../../stores/modules';
 
 	export let toggle = true;
 	export let slug: string;
+	export let isActive: boolean;
 
 	const dispatch = createEventDispatcher();
 
-	$: isModuleActive = $modulesState[ slug ].active;
-	$: isModuleAvailable = $modulesState[ slug ].available;
+	$: isModuleActive = isActive;
```

Comment:
isActive and isModuleActive is now a bit confused - both are used, both seem the same?

Reply:
Wow - thanks. Not sure why I did that... I've simplified the check in 2701f1ac3c94a06c15ee0629d76538d70a07cb34.